jest.config.js

```diff
@@ -48,6 +51,7 @@ module.exports = {
     '<rootDir>/app/scripts/lib/**/*.test.ts',
     '<rootDir>/app/scripts/lib/createRPCMethodTrackingMiddleware.test.js',
     '<rootDir>/app/scripts/migrations/*.test.js',
+    '<rootDir>/app/scripts/migrations/*.test.ts',
```

Comment:
An ancillary change, but this allows us to write migration tests in TypeScript rather than JavaScript.